src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -48,11 +76,22 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
                 global.location.href = httpReferrer;
               }}
             >
-              Return to your learning platform
+              Return to dashboard
             </Button>
           )}
+          {showHelpfulLink(failureReason) && (
+            <div>
+              <span className="executive-education-2u-error-link-description pr-2">
+                Helpful link:
+              </span>
+              <span className="executive-education-2u-error-link-box">
+                <ContactAdminMailto />
+              </span>
+            </div>
+          )}
         </Col>
       </Row>
+      <div className="exec-ed-error-background bg-light-200" />
```

Comment:
[clarification] Is this needed? Running this locally, it seems the `bg-light-200` comes from [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/ddad30269725b056ea4c6f2f933332b722b1002a/src/components/executive-education-2u/ExecutiveEducation2UPage.jsx#L92) instead. Any reason why this line (and the class name) can't be removed?